language-design.md

```diff
@@ -28,7 +77,33 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
-
+The most important feature of a well designed language in our opinion is
+understandability. Steele mentions that including too many features for
+specialized groups of users would "just be too much" [Steele, 2012]. This
+is in line with understandability because users don't need to learn as 
+many features. This is also in line with knowing the intended audience for
+a language and catering to their needs - Java is a general purpose language
+and therefore should not be too specialized for any one field. 
```

Comment:
I think a possibly interesting theoretical question would be whether different parts of a language count as different languages. Would a team using only features A-C of a language be considered using the same language as a team using only features D-F? Where are distinctions between dialects and languages?